ecommerce/serializers.py

```diff
@@ -172,7 +172,9 @@ def get_discounted_price(self, instance):
         discounts = instance.discounts.all()
 
         if discounts.count() == 0:
-            return None
+            return sum(
+                basket_item.product.price for basket_item in instance.basket_items.all()
```

Comment:
The only thing that concerns me here, is that in the discounted amount the the discounted price is being multiplied by the quantity attribute of BasketItem, and here it is just the price independently of quantity of items. Maybe let's make it consistent.

Reply:
Good call; I added a method to the BasketItems model to handle the base price (before discounts) of the item in https://github.com/mitodl/mitxonline/pull/472/commits/a5d67a2057a9ef96698896b90d553f53dc2170f8